modules/@apostrophecms/permission/index.js

```diff
@@ -175,7 +175,7 @@ module.exports = {
 
             return query;
           }
-        } else if (action === 'edit') {
+        } else if (action === 'edit' || action === 'create' || action === 'archive') {
```

Comment:
This may well be correct, but instead I recommend immediately changing `action` from `create` or `archive` to `edit` right at the top of both `can` and `criteria`, to minimize the need for other changes in those methods.